qbittorrent.json

```diff
@@ -4,11 +4,11 @@
     "version": "4.0.4",
     "architecture": {
         "64bit": {
-            "url": "https://sourceforge.net/projects/qbittorrent/files/qbittorrent-win32/qbittorrent-4.0.4/qbittorrent_4.0.4_x64_setup.exe#/dl.7z",
+            "url": "https://netix.dl.sourceforge.net/project/qbittorrent/qbittorrent-win32/qbittorrent-4.0.4/qbittorrent_4.0.4_x64_setup.exe#/dl.7z",
```

Comment:
I don't know if this is the right thing to do. This is hardcoding a specific server in sourceforge's rotation, which may go offline or not have the file even while other servers would still be available.

Reply:
True. I was wrong as well about the fosshub urls. Since it seems to be working again now I'll close this.